umm-spec-lib/resources/json-schemas/collection/umm/v1.10/umm-c-json-schema.json

```diff
@@ -1,6 +1,45 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
   "definitions": {
+    "UseConstraintsDescriptionType": {
```

Comment:
Why don't we make this a new version?

Reply:
I don't know. @eereiter?

Reply:
Because the customer wants to know what will be in a version up front, so I have defined what will be in each version.  So the versions are now defined by capability and not sprint time boundaries.